tests/parsers/pipe.parser.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('PipeParser', () => {
 	});
 
 	it('should extract strings with escaped quotes', () => {
-		const contents = `Hello {{ 'World\'s largest potato' | translate }}`;
+		const contents = `Hello {{ 'World\\'s largest potato' | translate }}`;
```

Comment:
This won't work. You're escaping the backslash, making the single quote stand alone, resulting in a syntax error (at least when using AOT). Single backslash + single quote, is the correct way to perform the escaping in templates. Any idea how we can fix this?

Please also add a test confirming the bug is actually fixed. This should do:

```ts
	it('should extract string using pipe, but between quotes only', () => {
		const contents = `<input class="form-control" type="text" placeholder="{{'user.settings.form.phone.placeholder' | translate}}" [formControl]="settingsForm.controls['phone']">`;
		const keys = parser.extract(contents, templateFilename).keys();
		expect(keys).to.deep.equal(['user.settings.form.phone.placeholder']);
	});
```